webapp/views/App/views/Designer/CategoryList/CategoryList.js

```diff
@@ -38,21 +35,25 @@ const CategoryList = (props) => {
         headerLeftComponent={TableHeaderLeft}
         rowHeaderComponent={TableRowHeader}
         rowComponent={TableRow}
-        headerProps={{ inCategoriesPath, canSelect, onSelect }}
-        rowProps={{ inCategoriesPath, canSelect, onSelect, selectedItemUuid }}
+        headerProps={{ listState: state }}
+        rowProps={{ listState: state, setListState: setState }}
```

Comment:
do you have local state in the rows? you can just name this state!

Reply:
in fact, state seems the local state of the row... are you sure I have to rename it?